osu.Game/Screens/Edit/Compose/Components/Timeline/TimelineBlueprintContainer.cs

```diff
@@ -31,22 +31,12 @@ internal class TimelineBlueprintContainer : EditorBlueprintContainer
         [Resolved(CanBeNull = true)]
         private Timeline timeline { get; set; }
 
-        [Resolved]
-        private OsuColour colours { get; set; }
-
         private DragEvent lastDragEvent;
         private Bindable<HitObject> placement;
         private SelectionBlueprint<HitObject> placementBlueprint;
 
-        private SelectableAreaBackground backgroundBox;
-
-        // we only care about checking vertical validity.
-        // this allows selecting and dragging selections before time=0.
-        public override bool ReceivePositionalInputAt(Vector2 screenSpacePos)
-        {
-            float localY = ToLocalSpace(screenSpacePos).Y;
-            return DrawRectangle.Top <= localY && DrawRectangle.Bottom >= localY;
-        }
+        // We want children to be able to be clicked and dragged, regardless of this drawable's size
+        public override bool ReceivePositionalInputAt(Vector2 screenSpacePos) => true;
```

Comment:
Everything else in this diff I suppose I could agree with, but this seems sort of excessive/heavy-handed. Surely the `TimelineBlueprintContainer` doesn't need to receive positional input outside the bounds of its parenting `Timeline`? The timeline is cached too, so in theory you could constrain this to the timeline's screen-space quad.

Not entirely sure if worth fixing, but this is the first biggest thing that jumps out of this diff to me.